README.md

```diff
@@ -37,7 +37,7 @@ There are three recommended patterns for running MaxText. You can run locally, r
 ## Getting Started: Download Dataset and Configure
 You need to run these steps once per project prior to any local development or cluster experiments.
 
-1. Create two gcs buckets in your project, one for to downloading and retreiving the dataset and the other for storing the logs.
+1. Create two gcs buckets in your project, one for to downloading and retrieving the dataset and the other for storing the logs.
```

Comment:
The hero we need <3 Probably better form to have this a separate PR (and maybe fix other typos), but obviously this is fine here.

Reply:
Glad to be helpful :D 

Hmm yes I was indecisive about splitting the changes since it was just a handful of type changes... but I agree on form and I'll split it up next time!